projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -483,6 +483,48 @@ export function canOpenWithOffice(context: AcaRuleContext): boolean {
   return context.permissions.check(file, ['update']);
 }
 
+/**
+ * Checks if user savedSearches are supported by current ACS version.
+ * JSON ref: `isSavedSearchAvailable`
+ */
+export const isSavedSearchAvailable = createVersionRule('25.1.0');
+
+/**
+ * Partially applies minimal version of a feature against a core compatibility evaluation.
+ * @param minimalVersion The minimal version to check against.
+ */
+export function createVersionRule(minimalVersion: string): (context: RuleContext) => boolean {
+  return (context: RuleContext): boolean => {
+    const acsVersion = context.repository.version?.display?.split(' ')[0];
+    return isVersionCompatible(acsVersion, minimalVersion);
+  };
+}
+
+export function isVersionCompatible(currentVersion: string, minimalVersion: string): boolean {
```

Comment:
no need to export and no need to document functions which are not used outside that file (are not public). If something is exported only because of access in unit tests then it should be tested indirectly through public functions when that private function is used